src/dom/getScale.js

```diff
@@ -0,0 +1,17 @@
+// @function getScale(el: HTMLElement): Object
```

Comment:
Docstrings are missing the namespace. Please add `// @namespace DomEvent` before.